### DISCUSSION TITLE
Hosting my remix-i18next integrated Remix app on Netlify

### DISCUSSION THREAD

**Discussion by wilsonbirch** [sergiodxa/remix-i18next#69] [Q&A] [answered]
Does anyone have experience with hosting their remix app on netlify once they have integrated i18n? I have tried multiple things to fix this but can't seem to figure out, anyone with any ideas/thoughts would be appreciated!
![Screenshot (192)](https://user-images.githubusercontent.com/63611537/170720054-781162e6-d50a-4e25-b594-8297c788c084.png)

**Comment by sergiodxa** [comment, marked as answer]
Looks like you’re trying to read the json files from disk? But on Netlify and other Serverless platforms access to the file system is not reliable.

Check what’s the actual final location of the json file or try to load it from the URL.

**Reply by wilsonbirch** [reply]
I have the locale folder located within the public folder in the root of my repo, should this be somewhere else?

**Reply by sergiodxa** [reply]
I think they move the public folder content somewhere else, so it may not be in public anymore
